main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -1349,6 +1365,10 @@ object Namer {
     case WeededAst.Pattern.Cst(Ast.Constant.Null, loc) => throw InternalCompilerException("unexpected null pattern", loc)
     case WeededAst.Pattern.Tag(qname, p, loc) => freeVars(p)
     case WeededAst.Pattern.Tuple(elms, loc) => elms flatMap freeVars
+    case WeededAst.Pattern.Record(pats, pat, _) =>
+      // pats.map(_.tpe.map(freeTypeVars)).filterNot(_.isEmpty).flatMap(_.get) ++ <---- include type vars?
```

Comment:
Should type vars be included in `freeVars`?

Reply:
@mlutze what do you think about this?

Reply:
No, only term vars.